indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -153,14 +153,6 @@
 
     <div class="toolbar f-j-end hide-if-locked">
         <div class="group">
-<<<<<<< HEAD
-=======
-            <a class="i-button" href="{{ url_for('.quick_setup_notifications', event) }}">
-                {% trans %}Quick setup{% endtrans %}
-            </a>
-        </div>
-        <div class="group">
->>>>>>> 420cc60e6b (Add quick setup to call for abstracts notifications)
```

Comment:
You left some conflict markers here. Could you also fix the current conflict? thanks :)

Reply:
Yeah, those are in the first commit but fixed it in the 2nd, do you want me to fix it? it won't be pushed anyway

Reply:
Hmm I was probably looking at an outdated diff, all good then, if you could just rebase :)

Reply:
Done :)